src/Microsoft.DotNet.Wpf/src/PresentationUI/PresentationUI.csproj

```diff
@@ -15,6 +13,7 @@
     <InternalMarkupCompilation>true</InternalMarkupCompilation>
     <NoInternalTypeHelper>true</NoInternalTypeHelper>
     <GenerateDependencyFile>false</GenerateDependencyFile>
+    <DoNotLimitMicrosoftNetCoreReferences>true</DoNotLimitMicrosoftNetCoreReferences>
```

Comment:
Why is this needed ? It looks like you removed the handling of this property in the .targets files and this is now the only use `DoNotLimitMicrosoftNetCoreReferences`.
```suggestion
```

Reply:
Missed it, thanks :)